default.nix

```diff
@@ -57,8 +57,8 @@ let
           fourmolu-src = pkgs.fetchFromGitHub {
             owner  = "parsonsmatt";
             repo   = "fourmolu";
-            rev    = "1de072e6bf916dbb1d7c8caaf1946dfeb5c4a161";
-            sha256 = "0fcins4yama37zd84jrhc2g9289jss3k1sn0y0cir13kmi5i0h1k";
+            rev    = "53a57a3bd7bec4beabff4612276365f07edcdd97";
```

Comment:
I'd probably switch this back to `callHackageDirect` since the [stack.yaml](https://github.com/haskell/haskell-language-server/blame/master/stack-8.6.5.yaml#L24) for 8.6.5 (which is the intended primary target for this repo) switched to that.